### ISSUE TITLE
Only the first node runs

### ISSUE THREAD

**Issue by themooer1** [revenz/FileFlows#437]
I have a simple flow which copies files from `/my/ripped/movies` to `/mnt/media/movies`.  It looks like this:

```
{
  "Enabled": true,
  "Description": null,
  "Type": 0,
  "Template": null,
  "Parts": [
    {
      "Uid": "e469b43a-7189-4df4-96a6-ec1602cc3cdf",
      "Name": "InputFile",
      "FlowElementUid": "InputFile",
      "xPos": 280,
      "yPos": 120,
      "Icon": "far fa-file",
      "Label": "",
      "Inputs": 0,
      "Outputs": 1,
      "OutputConnections": [
        {
          "Input": 1,
          "Output": 1,
          "InputNode": "165fc1fb-a594-487e-a6af-0ac52eac7f44"
        }
      ],
      "Type": 0,
      "Model": null
    },
    {
      "Uid": "165fc1fb-a594-487e-a6af-0ac52eac7f44",
      "Name": "Move to Plex",
      "FlowElementUid": "FileFlows.BasicNodes.File.MoveFile",
      "xPos": 280,
      "yPos": 340,
      "Icon": "fas fa-file-export",
      "Label": "",
      "Inputs": 1,
      "Outputs": 2,
      "OutputConnections": null,
      "Type": 2,
      "Model": {
        "DestinationPath": "/mnt/media/movies",
        "DestinationFile": "",
        "MoveFolder": false,
        "DeleteOriginal": false,
        "AdditionalFiles": [],
        "AdditionalFilesFromOriginal": false
      }
    }
  ],
  "Default": false,
  "Uid": "02d7adaa-6a3a-43d6-a3fb-7a4d610fcbfe",
  "Name": "Copy Movie Rips to Plex",
  "DateCreated": "2023-03-24T19:01:04.3916362-04:00",
  "DateModified": "2023-03-24T19:25:19.8749328-04:00"
}
```

However, when I run this against a library, the log says that only the InputFile node ran, the job completes instantly, and the ~32GB file is not copied.

```
==============================================================================
===                      PROCESSING NODE OUTPUT START                      ===
==============================================================================
2023-03-24 19:25:54.590 [INFO] -> Flow Runner Version: 1.0.9.2211
2023-03-24 19:25:54.621 [INFO] -> Using configuration encryption key: e862af13-78f8-4936-b27b-a3b063072010
2023-03-24 19:25:54.713 [INFO] -> Base URL: http://localhost:5000
2023-03-24 19:25:54.713 [INFO] -> Docker: True
2023-03-24 19:25:54.713 [INFO] -> Working Directory: /temp/Runner-16b2906a-38fe-4fad-b0b3-cb674a9db3de
2023-03-24 19:25:54.713 [INFO] -> Created Directory: /temp/Runner-16b2906a-38fe-4fad-b0b3-cb674a9db3de
2023-03-24 19:25:54.718 [INFO] -> Address: INTERNAL_NODE
2023-03-24 19:25:54.749 [INFO] -> Waiting on node task
2023-03-24 19:25:54.794 [INFO] -> Completed node task
2023-03-24 19:25:54.794 [INFO] -> Node SignalrUrl: http://localhost:5000/flow
2023-03-24 19:25:54.867 [INFO] -> Initial Size: 32954630585
2023-03-24 19:25:54.916 [INFO] -> SignalrUrl: http://localhost:5000/flow
2023-03-24 19:25:55.097 [INFO] -> File: /mnt/rips/video/completed/movies/Moneyball (2011)/Moneyball (2011).mkv
2023-03-24 19:25:55.102 [INFO] -> Executing Flow: Copy Movie Rips to Plex
2023-03-24 19:25:55.102 [INFO] -> Version: 1.0.9.2211
2023-03-24 19:25:55.102 [INFO] -> Platform: Docker
2023-03-24 19:25:55.272 [INFO] -> ======================================================================
2023-03-24 19:25:55.272 [INFO] -> Executing Node 1: InputFile [FileFlows.Plugin.Node]
2023-03-24 19:25:55.273 [INFO] -> ======================================================================
2023-03-24 19:25:55.274 [INFO] -> Node execution time: 00:00:00.0003671
2023-03-24 19:25:55.274 [INFO] -> ======================================================================
2023-03-24 19:25:55.547 [INFO] -> Final Fingerprint: db7c1987bd7c7a47d3c458013b3284a4778451096df194173624b03fcba9fd1a
2023-03-24 19:25:55.548 [INFO] -> Original Size: 32954630585
2023-03-24 19:25:55.548 [INFO] -> Final Size: 32954630585
2023-03-24 19:25:55.549 [INFO] -> Output Path: /mnt/rips/video/completed/movies/Moneyball (2011)/Moneyball (2011).mkv
2023-03-24 19:25:55.549 [INFO] -> Final Status: Processed
2023-03-24 19:26:00.175 [INFO] -> Connection_Closed
2023-03-24 19:26:00.435 [INFO] -> Final Fingerprint: db7c1987bd7c7a47d3c458013b3284a4778451096df194173624b03fcba9fd1a
2023-03-24 19:26:00.435 [INFO] -> Original Size: 32954630585
2023-03-24 19:26:00.435 [INFO] -> Final Size: 32954630585
2023-03-24 19:26:00.435 [INFO] -> Output Path: /mnt/rips/video/completed/movies/Moneyball (2011)/Moneyball (2011).mkv
2023-03-24 19:26:00.435 [INFO] -> Final Status: Processed
2023-03-24 19:26:00.436 [INFO] -> Making request [POST]: http://localhost:5000/api/worker/work/finish
2023-03-24 19:26:00.450 [INFO] -> Exit Code: 0

==============================================================================
===                       PROCESSING NODE OUTPUT END                       ===
==============================================================================
```

Thanks for taking a look!

**Comment by revenz** [comment]
not sure how this flow was created, but the first element is invalid.

Importing this flow, yes, it kinda works, but exits after the first node.

If i recreate the flow, then 
```
      "FlowElementUid": "InputFile",
...

becomes
```
      "FlowElementUid": "FileFlows.BasicNodes.File.InputFile"
```
and then it works.

So how was this flow created?

**Comment by themooer1** [comment]
I'm on fileflows 1.0.9_4.0.12

I created this flow multiple times in the web UI yesterday, and every time I got something non-functional.  Now, however, even though I was able to show in the video that the current, non working flow looks normal in the web gui, and exports with an invalid FlowElementUid property, when I recreated the flow and export that, the property was correct and the flow worked.

https://youtu.be/Qoft35PqTIo

The log of it now working:

```
/mnt/rips/video/completed/movies/Moneyball (2011)/Moneyball (2011).mkv
2023-03-25 22:16:16.132 [INFO] -> File: /mnt/rips/video/completed/movies/Moneyball (2011)/Moneyball (2011).mkv2023-03-25 22:16:16.137 [INFO] -> Executing Flow: Copy Movie Rips to Plex2023-03-25 22:16:16.137 [INFO] -> Version: 1.0.9.22112023-03-25 22:16:16.137 [INFO] -> Platform: Docker
2023-03-25 22:16:16.204 [INFO] -> ======================================================================
2023-03-25 22:16:16.204 [INFO] -> Executing Node 1: InputFile [FileFlows.BasicNodes.File.InputFile]2023-03-25 22:16:16.204 [INFO] -> ======================================================================
2023-03-25 22:16:16.206 [INFO] -> Node execution time: 00:00:00.0006110
2023-03-25 22:16:16.206 [INFO] -> ======================================================================2023-03-25 22:16:16.206 [DBUG] -> output: 1
2023-03-25 22:16:16.209 [INFO] -> ======================================================================
2023-03-25 22:16:16.209 [INFO] -> Executing Node 2: Move File [FileFlows.BasicNodes.File.MoveFile]2023-03-25 22:16:16.209 [INFO] -> ======================================================================
2023-03-25 22:16:16.250 [INFO] -> Creating folder: /mnt/media/movies/Moneyball (2011)
2023-03-25 22:16:16.348 [INFO] -> Changing owner on folder: /mnt/media/movies/Moneyball (2011)/
2023-03-25 22:16:16.349 [INFO] -> Change owner command: chown -R 3009:3004 /mnt/media/movies/Moneyball\ \(2011\)/
2023-03-25 22:16:16.385 [INFO] -> Setting permissions on folder: /mnt/media/movies/Moneyball (2011)/
2023-03-25 22:16:16.400 [INFO] -> About to move file to: /mnt/media/movies/Moneyball (2011)/Moneyball (2011).mkv
2023-03-25 22:16:16.401 [INFO] -> Mapped destination path: /mnt/media/movies/Moneyball (2011)/Moneyball (2011).mkv
2023-03-25 22:16:16.403 [INFO] -> Moving file: "/mnt/rips/video/completed/movies/Moneyball (2011)/Moneyball (2011).mkv" to "/mnt/media/movies/Moneyball (2011)/Moneyball (2011).mkv"
```

**Comment by revenz** [comment]
if you can come up with repo steps for how you made  a flow with the invalid elementuid, then I can repo it and fix it.

**Comment by crazybob1215** [comment]
I'm seeing the same issue, but I haven't been able to stumble into a working flow. 

Here's the export of the flow:
`
{
  "Enabled": true,
  "Description": "A blank Video template with a single \u0022Video File\u0022 input node.",
  "Type": 0,
  "Template": "Video File",
  "Parts": [
    {
      "Uid": "408358e1-9ab1-4307-859a-0b1595cf5274",
      "Name": "",
      "FlowElementUid": "FileFlows.VideoNodes.VideoFile",
      "xPos": 450,
      "yPos": 50,
      "Icon": "fas fa-video",
      "Label": "",
      "Inputs": 0,
      "Outputs": 1,
      "OutputConnections": [
        {
          "Input": 1,
          "Output": 1,
          "InputNode": "a9e9d80a-8318-49f6-be78-fa755e2d218e"
        }
      ],
      "Type": 0,
      "Model": null
    },
    {
      "Uid": "a9e9d80a-8318-49f6-be78-fa755e2d218e",
      "Name": "",
      "FlowElementUid": "FileFlows.BasicNodes.Functions.Log",
      "xPos": 450,
      "yPos": 180,
      "Icon": "far fa-file-alt",
      "Label": "",
      "Inputs": 1,
      "Outputs": 1,
      "OutputConnections": [
        {
          "Input": 1,
          "Output": 1,
          "InputNode": "79606582-6127-4f74-8b02-ef2d66f427c6"
        }
      ],
      "Type": 3,
      "Model": {
        "LogType": 2,
        "Message": "Yahaha! You found me!"
      }
    },
    {
      "Uid": "79606582-6127-4f74-8b02-ef2d66f427c6",
      "Name": "",
      "FlowElementUid": "FileFlows.BasicNodes.File.Touch",
      "xPos": 450,
      "yPos": 290,
      "Icon": "fas fa-hand-point-right",
      "Label": "",
      "Inputs": 1,
      "Outputs": 1,
      "OutputConnections": [
        {
          "Input": 1,
          "Output": 1,
          "InputNode": "93772f7e-1cd1-40d3-b76c-358b6413cc00"
        }
      ],
      "Type": 2,
      "Model": {
        "FileName": "/temp/test.txt"
      }
    },
    {
      "Uid": "93772f7e-1cd1-40d3-b76c-358b6413cc00",
      "Name": "",
      "FlowElementUid": "FileFlows.BasicNodes.File.CopyFile",
      "xPos": 450,
      "yPos": 400,
      "Icon": "far fa-copy",
      "Label": "Copy File",
      "Inputs": 1,
      "Outputs": 1,
      "OutputConnections": null,
      "Type": 2,
      "Model": {
        "DestinationPath": "/temp",
        "DestinationFile": "{file.NameNoExtension}_test{file.Extension}",
        "CopyFolder": false,
        "AdditionalFiles": null,
        "AdditionalFilesFromOriginal": false
      }
    }
  ],
  "Default": false,
  "Uid": "9295b034-e26a-4c85-b02c-cb5c7e324aab",
  "Name": "testLoad",
  "DateCreated": "2023-04-04T12:22:34.6594777Z",
  "DateModified": "2023-04-04T16:24:18.9691526Z"
}
`

Here's the log output:
![image](https://user-images.githubusercontent.com/45743617/229913136-ca20e728-e934-4c6b-9468-a92d50b72c1c.png)

Here are the steps I followed to create this flow:
1. In the flows view, click "+ ADD"
2. Gave the flow the name "testLoad"
3. Selected the "Video File" template
4. Added a "Log" node, changed the type to "information" and added the message
5. Connected the "Video File" node to the "Log" node
6. Added a "Touch" node, specified the path and destination file
7. Saved the flow
8. Moved to the "Libraries" view
9. Edited the current library in use to use the newly created "testLoad" flow
10. In the case of the above log, I wen to the "Files" view, selected a processed file and clicked "Reprocess". I've also tried importing a brand new file to the watched directory and I get the same result.

**Comment by revenz** [comment]
Your issue appears different, your flow is fine, I just imported it and redid your repo steps.  But your log is shown it couldnt find the video file node and used the default "FileFlows.Plugin.Node" instead.   I just updated 1.1.1 to throw an error instead of return the default so if this happens again, the flow will fail.

This suggests it can't read VideoNodes dll from the configuration directory for some reason

**Comment by crazybob1215** [comment]
That's odd. I stood my instance up using the default docker-compose, with just the paths and timezone changed, so I'm not sure why it would be missing a config item. Is there something I can do to try and repair the missing dll?

I just tried deleting the Video Node plugin, restarting the container, reinstalling the plugin and re-processing the file, but I'm getting the same result. Also, should I split this out into it's own issue since you said it doesn't match what this issue was opened for?

**Comment by revenz** [comment]
yeah best to split it
